sentinel-core/src/main/java/com/alibaba/csp/sentinel/log/LogTarget.java

```diff
@@ -0,0 +1,13 @@
+package com.alibaba.csp.sentinel.log;
+
+import java.lang.annotation.*;
+
+/**
+ * @author xue8
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+@Documented
+public @interface LogTarget {
+    LogType value() default LogType.RECORD_LOG;
```

Comment:
Ok, i will do it as soon as possible.

Reply:
> Because types in `LogType` share the same interface is it necessary enabling users specify the same implementation and merging all the logs?
> 
> Something like `LogType[] value()` and new type `ALL` in `LogType`.

Because the `log` and `logger` are one-to-one`, if all the logs use the same implementation, they will be using the same `logger`, then it is impossible to configure the logger configuration different in `XML`.

Reply:
> > Because types in `LogType` share the same interface is it necessary enabling users specify the same implementation and merging all the logs?
> > Something like `LogType[] value()` and new type `ALL` in `LogType`.
> 
> Because the `log` and `logger` are one-to-one, if all the logs use the same implementation, they will be using the same `logger`, then it is impossible to configure the logger configuration different in `XML`.

Okay i know. So they are stateful. That's fine.